src/ethereum.js

```diff
@@ -0,0 +1,41 @@
+require('babel-polyfill')
+require('dotenv').config({path: './.env'})
+const { eth, contracts } = require('decentraland-commons')
+const web3Eth = eth
+const { LANDRegistry } = contracts
+
+async function connectBlockchain () {
+  try {
+    await web3Eth.disconnect() // clean if it is a retry
+    let connected = await web3Eth.connect({
+      contracts: [LANDRegistry]
+    })
+    if (!connected) {
+      throw new Error('Could not connect to the blockchain')
+    }
+  } catch (e) {
+    if (e.message.indexOf('Could not connect to the blockchain') !== -1) {
```

Comment:
if this statement necessary? what would be another case where we wouldnt require to try to re-connect?

Reply:
Nope, it was a copy & paste from another project